R/catmaid_connection.R

```diff
@@ -129,6 +130,9 @@
 #'   config=conn$config)
 #' }
 #' @export
+#' 
+#' @examples
+#' conn=catmaid_login()
```

Comment:
Please don't add an unprotected example like this. It is not guaranteed that an end user will be able to call catmaid_login without any arguments since the may not have setup .Renviron or .Rprofile appropriately.

Reply:
I have removed this in the latest commit, that being said the reason I added this is that the bug is reproducible only if you have an example running. So we need to have such an example in the future.